Client avoidance of duplicate names produces duplicate files

After this PR : https://github.com/Azure/autorest.go/commit/3458098afd88db38c357921cbd964aa72685a61d  

there is an unexpected behavior in the generation for example: ` azure-rest-api-specs\specification\automation `

It results in creating duplicate files with the same content the only difference is that one has the extra `Group` suffix.  ( **runbookdraft.go**, **runbookdraftgroup.go** )
 
As a side effect the build failures because teh future names are getting updated to include the `Group` part although they are referenced in the **runbookdraft** files .

```
.\runbookdraft.go:49:158: undefined: RunbookDraftCreateOrUpdateFuture
.\runbookdraft.go:96:82: undefined: RunbookDraftCreateOrUpdateFuture
.\runbookdraft.go:270:121: undefined: RunbookDraftPublishFuture
.\runbookdraft.go:316:75: undefined: RunbookDraftPublishFuture
```

--- a/specification/automation/resource-manager/readme.md
+++ b/specification/automation/resource-manager/readme.md
@@ -162,7 +162,7 @@ go:
 
 ### Go multi-api
 
-``` yaml $(go)
+``` yaml $(go) && $(multiapi)
 batch:
   - tag: package-2015-10
   - tag: package-2017-05-preview

